models/channel/src/channel-draft/presets/sample-data-fashion/store-other.ts

```diff
@@ -0,0 +1,15 @@
+import { LocalizedString } from '@commercetools-test-data/commons';
+import { roles } from '../../../constants';
+import ChannelDraft from '../../builder';
+
+const storeOther = () =>
+  ChannelDraft()
+    .key('store_other')
+    .address(undefined)
+    .geoLocation(undefined)
+    .custom(undefined)
```

Comment:
Not needed

Reply:
I might have been thinking too much about implementation here, is the plan to use these in the resolvers like this:
```
import presets from '../presets'

const storeUsa = presets.SampleDataFashion.storeUsa;

// Ignore this obviously wrong syntax, it's just to illustrate a point
makeRequestToCoCo('url', {
    variables: {
        draft: {
            name: storeUsa.name,
            description: storeUsa.description,
            key: storeUsa.key,
            roles: storeUsa.roles,
        }
    }
})```

or like this

```
import presets from '../presets'

makeRequestToCoCo('url', {
    variables: {
        draft: { ...presets.SampleDataFashion.storeUsa() }
    }
})
```

I was thinking of the latter, hence my obsession with ensuring the presets were of a specific shape - but that might be too obsessive 😬

Reply:
Definitely the latter! Those three undefined fields aren't needed because they aren't required fields and we aren't using them.

Reply:
Ohhhh. I'm picking up what you're putting down. We just won't include these fields in the query, so they will be ignored.